README.md

```diff
@@ -52,6 +64,14 @@ message without internet connection? No problem, the message is still there.
 GNU General Public License v2.
 
 
+## Why should I use OfflineIMAP?
+
+* It is **fast**.
+* It is **reliable**.
+* It is **flexible**.
+* It is **safe**.
+
+
```

Comment:
Why these changes? Please, see a92b116a4f0b70396049deca779783309de8d897

Reply:
Those "it is flexible it is safe" lines came in by mistake, because, added the new build status icon badges to the 7.1.13 version of `README.md`. Those lines can be removed.

Reply:
Lines removed.